### ISSUE TITLE
Landing Autopilot Isn't Working

### ISSUE THREAD

**Issue by arlaurent** [MuMech/MechJeb2#1362]
I'm on 1.9.1 and running the latest release of mechjeb. When I click Landing Guidance at the mun with a target or "land somewhere", nothing happens. The vehicle orients itself. It warps to the right position, but ignition and burns do not occur. I am not quite sure what the bug is or how to document it, so any advice would be appreciated.

My mods,

```
000_ClickThroughBlocker 
001_ToolbarControl 
B9PartSwitch
 Chatterer 
CommunityCategoryKit 
CommunityResourcePack 
CommunityTechTree 
ContractConfigurator 
ContractPacks 
CryoEngines 
CryoEnginesNFAero 
CryoEnginesRestock 
CryoTanks 
DMagicOrbitalScience 
DecayingRTGs 
DeployableEngines 
DynamicBatteryStorage 
HeatControl 
HideEmptyTechTreeNodes 
KAS 
KIS 
KSPRescuePodFix 
KerbalAtomics 
KerbalAtomicsLH2NTRModSupport 
KerbalEngineer 
Kerbalism 
KerbalismConfig 
KronalVesselViewer 
MarkIVSystem 
MechJeb2 
ModuleManager.4.1.4.dll 
ModuleManager.ConfigCache 
ModuleManager.ConfigSHA 
ModuleManager.Physics 
ModuleManager.TechTree 
NearFutureAeronautics 
NearFutureConstruction 
NearFutureElectricaNTRs 
NearFutureElectrical 
NearFutureExploration 
NearFutureLaunchVehicles 
NearFutureMethalox 
NearFutureProps 
NearFuturePropulsion 
NearFutureSolar 
NearFutureSpacecraft 
ReStock 
ReStockPlus 
RealPlume 
RealPlume-Stock 
RestockRigidLegs 
SCANsat 
SmokeScreen 
SpaceTuxLibrary 
Squad 
SquadExpansion 
StationPartsExpansionMetal 
StationPartsExpansionRedux 
TriggerTech 
UniversalStorage2 
WaypointManager 
XenonHallThrusters
```

**Comment by arlaurent** [comment]
Further notes, minor things like "Deploy Solar Panels" and antennas aren't working for ascent guidance either. Hrm.

**Comment by sarbian** [comment]
The error you are reporting is fixed in the last MJ (2.11.0) . You did not include a log so I can not know which version of MJ you are running or if you have an other problem. I am closing this. If you still have that problem with 2.11 then re open it with a log.
